metadata.display.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
Was this updated manually or by regenerating the docs? If manually, please revert. No need to update license dates for existing files.

Reply:
It was automatically generated by the docs.